src/test/ui/parser/recover-assoc-eq-missing-term.stderr

```diff
@@ -0,0 +1,17 @@
+error: missing type to the right of `=`
+  --> $DIR/recover-assoc-eq-missing-term.rs:3:11
+   |
+LL |     bar::<Item = >();
+   |           ^^^^^^
+   |
+help: to constrain the associated type, add a type after `=`
+   |
+LL |     bar::<Item = TheType >();
+   |           ^^^^^^^^^^^^^^
+help: remove the `=` if `Item` is a type
+   |
+LL |     bar::<Item  >();
+   |               --
```

Comment:
Can we remove the whitespace too?

Reply:
Did the best I could, almost there, but not quite. Span hackery is deeply mysterious.

Reply:
Don't go overboard with span wrangling. It's finicky and ICE prone. We can rely on rustfmt for the last 20%.

Reply:
Yeah, @estebank I think that's good advice :)